test/functional/find-e2e-specs.js

```diff
@@ -10,7 +10,7 @@ chai.use(chaiAsPromised);
 
 const CAPS = {
   platformName: 'mac',
-  bundleId: TEXT_EDIT_BUNDLE_ID,
+  'appium:bundleId': TEXT_EDIT_BUNDLE_ID,
```

Comment:
what's this?

Reply:
this is a valid capability name for this driver. Check https://github.com/appium/appium-mac2-driver#capabilities